technical-reports/index.html

```diff
@@ -65,7 +66,7 @@ <h1>Modules</h1>
         <li>
           <a href="./format/">Format</a>
         </li>
-        <li>Color (coming soon)</li>
+        <li><a href="./color/">Color</a></li>
         <li>Animations (coming soon)</li>
```

Comment:
Unknown how best to update this (related to [@c1rrus’ comment](https://github.com/design-tokens/community-group/pull/147/files#r1392453903)), but perhaps we consolidate everything into `format/types.md`?